sonar-check-api/src/main/java/org/sonar/check/RuleProperty.java

```diff
@@ -26,7 +26,9 @@
 
 /**
  * @since 2.3
+ * @deprecated since 5.6 sonar-check-api is deprecated 
  */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 public @interface RuleProperty {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)